diff --git a/minimalstdlib/include/__memory_resource/lockfree_single_block_resource.h b/minimalstdlib/include/__memory_resource/lockfree_single_block_resource.h
index 21a1f19..e41cbec 100644
--- a/minimalstdlib/include/__memory_resource/lockfree_single_block_resource.h
+++ b/minimalstdlib/include/__memory_resource/lockfree_single_block_resource.h
@@ -19,7 +19,7 @@
 #include "lockfree/tagged_ptr"
 #include "memory_resource.h"
 
-#include "stdio.h"
+#include "minimalstdio.h"
 
 namespace MINIMAL_STD_NAMESPACE
 {
